content/docs/command-reference/metrics/index.md

```diff
@@ -63,10 +63,15 @@ stages:
           cache: false
 ```
 
-> `cache: false` above specifies that `summary.json` is not tracked or
-> <abbr>cached</abbr> by DVC (`-M` option of `dvc stage add`). These metrics
-> files are normally committed with Git instead. See `dvc.yaml` for more
-> information on the file format above.
+> Notes:
+>
+> - `cache: false` above specifies that `summary.json` is not tracked or
+>   <abbr>cached</abbr> by DVC (`-M` option of `dvc stage add`). These metrics
+>   files are normally committed with Git instead. See `dvc.yaml` for more
+>   information on the file format above.
+> - even though `dvc metrics` subcommands use the metrics files specifed in
+>   `dvc.yaml` by default, they can be used on any file, as long as they are
+>   specified via `targets` argument
```

Comment:
Hmmmm. I see. I think instead of changing this note we should incorporate the information to the paragraph before the `summary.json` sample block.

Reply:
I.e. instead of 

```markdown
`dvc metrics` subcommands by default use the metrics files specified in
`dvc.yaml` ...
```

We should have something like

```markdown

`dvc metrics` subcommands can be used on any
[valid metrics files](#supported-file-formats). By default they use the one
specified in `dvc.yaml` ...
```